apps/cookbook/src/app/showcase/badge-showcase/badge-showcase.component.html

```diff
@@ -1,8 +1,67 @@
-<div class="example">
-  <cookbook-badge-example></cookbook-badge-example>
-  <cookbook-code-viewer [html]="exampleHtml"></cookbook-code-viewer>
-  <h4>Properties:</h4>
-  <cookbook-api-description-properties
-    [properties]="properties"
-  ></cookbook-api-description-properties>
-</div>
+<p>
+  Badges are inlines block elements that often are used on top of or close to another component.
+  They can be used with numbers, text and icons. A small badge version also exists which does not
+  support slotted content but can be used to indicate status.
+</p>
+
+<kirby-divider [hasMargin]="true"></kirby-divider>
+
+<h2>Examples</h2>
+<h3>Badge with numbers</h3>
+<p>
+  Using a badge with an number can be a good way to indicate status on an element.
+</p>
+<p>
+  <kirby-card [mode]="_cardMode" [hasPadding]="_cardHasPadding">
+    <cookbook-badge-example-number #badgeExampleNumber></cookbook-badge-example-number>
+    <cookbook-example-viewer [html]="badgeExampleNumber.template"> </cookbook-example-viewer>
+  </kirby-card>
+</p>
+
+<h3>Badge with text</h3>
+<p>
+  Badges can also be used together with a text. Flags might in some cases be a better alternative.
+</p>
+<p>
+  <kirby-card [mode]="_cardMode" [hasPadding]="_cardHasPadding">
+    <cookbook-badge-example-text #badgeExampleText></cookbook-badge-example-text>
+    <cookbook-example-viewer [html]="badgeExampleText.template"> </cookbook-example-viewer>
+  </kirby-card>
+</p>
+
+<h3>Badge with icons</h3>
+<p>
+  It is possible to slot an icon to display it in a badge. This might be used to indicate the
+  existence of an associated element.
```

Comment:
```suggestion
  existence of an associated element (e.g. an attachment).
```

Consider adding this from the guidelines, to make it a little clearer.

Reply:
done!